pkg/controller/networkpolicy/networkpolicy_controller_test.go

```diff
@@ -110,6 +110,9 @@ func newController(objects ...runtime.Object) (*fake.Clientset, *networkPolicyCo
 	npController.cnpListerSynced = alwaysReady
 	npController.tierLister = crdInformerFactory.Security().V1alpha1().Tiers().Lister()
 	npController.tierListerSynced = alwaysReady
+	npController.cgInformer = crdInformerFactory.Core().V1alpha2().ClusterGroups()
```

Comment:
you could reuse cgInformer here and on L102

Reply:
Done